File: JavaScript/ArithmeticApp.js
let name = prompt("Enter your name:");
console.log("Welcome " + name);

console.log("========== Subtraction Game ==========");

let score = 0;

for (let count = 0; count < 10; count++) {

    let firstNumber = Math.floor(Math.random() * 70) + 50; 
    let secondNumber = Math.floor(Math.random() * 50) + 1; 

    let correctAnswer = firstNumber - secondNumber;

    let result = Number(prompt(firstNumber + " - " + secondNumber + " = "));

    if (result !== correctAnswer) {
        result = Number(prompt("Wrong! Try again:"));

        if (result === correctAnswer) {
            console.log("Correct!");
            score++;
        } else {
            console.log("Wrong again. Answer is: " + correctAnswer);
        }
    } else {
        console.log("Correct!");
        score++;
    }
}

console.log("Your total score is " + score + " over 10");

File: Python/Simple Arithmetic App/arithmetic_app.py
import random

name = input("Enter your name: ")
print("Welcome " + name)
print("========== Subtraction Game ==========")

score = 0

for number in range(1, 11):

    first_number = random.randint(50, 119)
    second_number = random.randint(1, 50)

    correct_answer = first_number - second_number

    print(first_number, "-", second_number, "=")
    
    result = int(input("Enter your answer: "))

    if result != correct_answer:
        result = int(input("Wrong! You have one more attempt: "))
        
        if result == correct_answer:
            print("Correct!")
            score += 1
  
print("Your final score is:", score)
